docker-compose.yml

```diff
@@ -854,6 +854,76 @@ services:
       - "traefik.http.routers.faf-nodebb.entryPoints=web-secure"
       - "traefik.http.routers.faf-nodebb.middlewares=test-compress@file"
       - "traefik.http.routers.faf-nodebb.tls.certresolver=default"
+
+  #
+  # FAF League Service
+  #
+  faf-league-service:
+    container_name: faf-league-service
+    image: cleborys/faf-league-service
+    user: ${FAF_LEAGUE_SERVICE_USER}
+    networks:
+      faf:
+        aliases:
+          - "faf-league-service"
+    restart: unless-stopped
+    env_file: ./config/faf-league-service/faf-league-service.env
+    depends_on:
+      - faf-init-volumes
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "${DEFAULT_LOG_SIZE}"
+        max-file: "${HIGH_PRIO_LOG_FILES}"
+
+  #
+  # FAF League Service MySQL database.
+  #
+  faf-league-db:
+    container_name: faf-league-db
+    image: mysql:5.7
+    user: ${FAF_LEAGUE_DB_USER}
+    networks:
+      faf:
+        aliases:
+          - "faf-league-db"
+    restart: unless-stopped
+    env_file: ./config/faf-league-db/faf-league-db.env
+    ports:
+        - "127.0.0.1:3307:3306"
```

Comment:
Just like the core database the league db is exposed to outside the `faf` network. The testing setup of the service depends on that (just like the python server), but if necessary I can think about how to make it work without exposing.